postgres/defaults.yaml

```diff
@@ -57,6 +57,10 @@ postgres:
 
   bake_image: False
 
+  fromrepo:
+  pkg_repo:
+    name: []
```

Comment:
This is a little bit dangerous and I would recommend to avoid setting such defaults. I mean `pkg_repo` dictionary.
1. Empty name is meaningless (why the list by the way?) and still will produce sneaky errors if other maps would not set it right.
2. If `use_upstream_repo` Pillar is set, the `upstream` state would always try to provision some repo config, even if that is not applicable for some systems.

Reply:
Thanks for the assessment -  the `from_repo` will work for both upstream and distro postgres. 
 do you think this stanza is unnecessary in defaults.yaml? Thanks.

Reply:
Only the `pkg_repo` is harmful, because it would interfere with repository config handling logic.
Keeping the `fromrepo` is safe and necessary though.